src/components/ProjectCreation.js

```diff
@@ -1,152 +1,117 @@
-                              onChange = {(e) => (setSkills(e.target.value))}
-                />
-             </Form.Group>
+    return (
+        <>
+            <div className="project-add-form">
+                <div className="name-description-container" style={{ flex: 1 }}>
+                    <div className="form-group">
+                        <label for="name">Name</label>
+                        <input className="form-control"
+                            id="name"
+                            type="text"
+                            placeholder="Enter Project Title"
+                            onChange={(e) => (setName(e.target.value))}
+                        />
+                    </div>
+                    <div className="form-group" style={{ flex: 1 }}>
+                        <label for="description">Description</label>
+                        <textarea className="form-control"
+                            id="description"
+                            style={{ height: '100%' }}
+                            type="text"
+                            placeholder="Enter Project Description"
+                            onChange={(e) => (setDescription(e.target.value))}
+                        />
+                    </div>
+                </div>
+                <div className="form-group" style={{ flex: 1 }}>
+                    <label for="tags">Tags</label>
+                    <textarea className="form-control"
+                        style={{ height: '100%' }}
+                        id="tags"
+                        type="text"
+                        placeholder="Enter Project Tags"
+                        onChange={(e) => (setTags(e.target.value))}
+                    />
+                </div>
+                <div className="form-group" style={{ flex: 1 }}>
+                    <label for="skills">Skills</label>
+                    <textarea className="form-control"
+                        style={{ height: '100%' }}
+                        id="skills"
+                        type="text"
+                        placeholder="Enter Project Skills"
+                        onChange={(e) => (setSkills(e.target.value))}
+                    />
+                </div>
+                <div className="form-group" style={{ flex: 1 }}>
+                    <label for="links">Links</label>
+                    <textarea className="form-control"
+                        style={{ height: '100%' }}
+                        id="links"
+                        type="text"
+                        placeholder="Enter Project Links"
+                        onChange={(e) => (setLinks(e.target.value))}
+                    />
+                </div>
+            </div>
+            <div style={{ textAlign: 'center', marginTop: '36px' }}>
+                <button className="btn btn-primary form-element" onClick={projectClick} style={{ padding: '8px 16px' }}>Submit</button>
```

Comment:
The `projectClick` function could have been renamed `createProject` or something similar